packages/test-base/src/jvmTest/kotlin/io/realm/kotlin/test/compiler/ModelDefinitionTests.kt

```diff
@@ -81,4 +81,20 @@ class ModelDefinitionTests {
         assertEquals(KotlinCompilation.ExitCode.INTERNAL_ERROR, result.exitCode, "Compilation should fail when using data class")
         assertTrue(result.messages.contains("Data class 'Foo' is not currently supported"))
     }
+
+    @Test
+    fun `enum_class`() {
```

Comment:
Not sure if I should name this differently

Reply:
Looks fine to me 👍